### PR TITLE
[MM-30264] Increase test timeout

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
Increases the timeout of the tests to avoid CI failures

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-30264

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -85,21 +85,21 @@ test: test-unit
 .PHONY: test-unit
 test-unit:
 	@echo Running unit tests
-	$(GO) test -mod=vendor -race -v -tags unit $(GO_PACKAGES)
+	$(GO) test -mod=vendor -timeout 30m -race -v -tags unit $(GO_PACKAGES)
```
Comment (hanzei) [inline Makefile:88]: Is it worth extracting these flags into a a separate variable to DRY the code?

**Review by hanzei — APPROVED** [review]
One non blocking question. LGTM :+1:

File: Makefile (hunk 1 above)
Comment (mgdelacroix) [reply Makefile]: Make sense, added in bd8425b

**Review by jespino — APPROVED** [review]
LGTM

**Pull request merged**